bluetooth/src/commonMain/kotlin/device/DefaultDeviceConnectionManager.kt

```diff
@@ -74,56 +59,104 @@ abstract class BaseDeviceConnectionManager(
         data class MtuUpdated(val newMtu: Int) : Event()
     }
 
+    val events: Flow<Event>
+    val rssi: Flow<Int>
+
+    fun getCurrentState(): State
+    suspend fun connect()
+    suspend fun discoverServices()
+    suspend fun disconnect()
+    suspend fun readRssi()
+    suspend fun requestMtu(mtu: Int): Boolean
+    suspend fun performAction(action: DeviceAction)
+    fun startConnecting()
+    fun cancelConnecting()
+    fun handleConnect()
+    fun startDiscovering()
+    fun startDisconnecting()
+    fun handleDisconnect(onDisconnect: (suspend () -> Unit)? = null)
+    suspend fun reset()
+    suspend fun unpair()
+    suspend fun pair()
+}
+
+abstract class BaseDeviceConnectionManager(
+    val deviceWrapper: DeviceWrapper,
+    private val settings: ConnectionSettings,
+    private val coroutineScope: CoroutineScope
+) : DeviceConnectionManager, CoroutineScope by coroutineScope {
+
+    internal companion object {
+        const val BUFFER_CAPACITY = 256
```

Comment:
EVENTS_BUFFER_SIZE? Or if it's unused since we switched to Channel

Reply:
now unused, removed in bd47cc4